src/components/Toaster/CountdownButton/index.test.tsx

```diff
@@ -1,19 +1,26 @@
 import React from 'react';
-import { shallow, mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 
-import CountdownButton, { CountdownButtonProps, getCircleInfo, CircleInfo } from '.';
-
-const renderComponent = (props: CountdownButtonProps) =>
-  shallow(<CountdownButton {...props} />);
+import CountdownButton, { getCircleInfo, CircleInfo } from '.';
 
 const MOCK_PROPS = { duration: 666 };
 
 describe('CountdownButton', () => {
-  it('matches snapshot', () => {
-    const component = renderComponent(MOCK_PROPS);
-    expect(component).toMatchSnapshot();
+  it('renders component', () => {
+    render(<CountdownButton {...MOCK_PROPS} />);
+    expect(screen.getByTitle('Close')).toBeInTheDocument();
   });
 
+  describe('CountdownButton animation', () => {
+    it('shows a full timer stroke on mount', () => {
+      const { container } = render(<CountdownButton {...MOCK_PROPS} />);
+      const circle = container.querySelector('div > div > svg > circle');
```

Comment:
My perspective is that this is probably against the principles of React Testing Library. We ideally want to test outcomes, not the makeup of the DOM itself. I would try a different approach (or happy to brainstorm with you a bit here).

Reply:
true, 100% right what you said, we could probably pair program, there's a few things i couldn't totally grasp about the testing of animations, or maybe i'm even making it a much bigger problem than it is, but still i think could benefit from a pairing session.

Reply:
Totally. Feel free to add something on my calendar for whenever you'd like.